chart/templates/operator.yaml

```diff
@@ -288,40 +287,54 @@ spec:
       labels:
         app: status-controller
     spec:
+      securityContext:
+        runAsNonRoot: true
+      serviceAccountName: status-sa
+      initContainers:
+      - name: create-crd
+        image: quay.io/kubestellar/kubectl:1.29.3
+        command: ['sh', '-c']
+        args:
+        - |
+          echo -n "Waiting for managedclusteraddons CRD"
+          while [ "$(kubectl get crd clustermanagementaddons.addon.open-cluster-management.io 2> /dev/null)" == "" ] ; do
+            echo -n "."
+            sleep 1
+          done
+          echo -e "\033[0;32m\xE2\x9C\x94\033[0m"
+          echo "Creating addon-status CR:"
```

Comment:
I would s/CR/object/.
Or s/Creating addon-status CR/Creating ClusterManagementAddOn addon-status/ .

BTW, why does this addon have "addon" in its name? Seems a bit redundant to me. Would something like "status-return" be more informative?

Reply:
I changed to object

Reply:
.. and I see that the "addon-" prefix comes from some common bit of kustomization.